libraries/AP_LQR_Control/AP_LQR_Control.h

```diff
@@ -98,14 +98,8 @@ class AP_LQR_Control : public AP_Navigation {
     // q2 multiplicative factor
     AP_Int16 _q2_val;
 
-    // cross track error factor
-    AP_Int16 _xtrack_fac;
-
-    // cross track error velocity factor
-    AP_Int16 _vel_fac;
-
-    // maximum allowed lateral acceleration
-    AP_Int16 _max_latacc;
+    // q1 multiplicative factor
+    AP_Int16 _q1_val;
```

Comment:
q1 is not needed. it is dynamically calculated ![image](https://user-images.githubusercontent.com/7996972/50403298-2c420300-07a6-11e9-9cfd-8ff54ef889b4.png)

Reply:
Made changes to allow user to use non adaptive gains also. If set to 0, adaptive gains are used instead.

Reply:
Great!